src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,60 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie,
+};
+
+export const MovieCard: React.FC<Props> = (
+  {
+    movie: {
+      description,
+      imdbUrl,
+      title,
+      imgUrl,
+    },
+  },
+) => (
```

Comment:
this looks like waaaay to much for a single param in a fuction, let's destructure `movie` itself` inside the component
```suggestion
export const MovieCard: React.FC<Props> = ({movie}) => (
```